Cargo.toml

```diff
@@ -37,14 +37,14 @@ document-features = "0.2"
 tracing = "0.1"
 tracing-subscriber = "0.3"
 
-# Macro utils
+# Utilities
+nevermore = "0.1"
```

Comment:
Should be `0.1.1`, taken the zeros out of the equation.

Reply:
Fair enough, will fix it tomorrow

Reply:
Done